scripts/ubuntu-bartender/ubuntu-bartender

```diff
@@ -262,6 +269,12 @@ build-provider-create $bartender_name
     cp -aR "$LIVECD_ROOTFS_DIR" $temp_dir/livecd-rootfs
   fi
 
+  if [ -n "$CHROOT_ARCHIVE" ]
+  then
+    cp -aR "$CHROOT_ARCHIVE" $temp_dir/chroot-archive
+    chroot_archive_flag="--use-chroot-archive ../chroot-archive"
```

Comment:
@davidkrauser Why do we copy to $temp_dir but then set the flag to use ../?

Also, does this get copied by old-fashioned-image-build into the chroot cache and used again if the user specifies --use-chroot-cache?  If so, that could be confusing.

Reply:
@rcj4747 bartender creates a directory structure like the following in the temp dir:

```
/livecd-rootfs (copied from livecd-rootfs git repo or local dir)
/livecd-rootfs/.../hooks/ (copied from extras git repo or local dir)
/ubuntu-old-fashioned (copied from git repo or local dir)
/chroot-archive (copied when flag is specified)
```

Bartender tars that up, and splats it into the home directory on the build vm. Then, it executes an old-fashioned image build from within the `/iivecd-rootfs` folder. From that location, the chroot archive is at `../chroot-archive`.

Old-fashioned does not copy the specified chroot archive into the /tmp dir normally used. If the user tries to run an old-fashioned image build after the initial run, and they don't specify a chroot archive the second time, old-fashioned will pull down a new one.

Reply:
Thank you.